tests/fixtures/ember-container-query-typescript/output/ember-container-query/package.json

```diff
@@ -36,10 +38,12 @@
     "lint:hbs:fix": "ember-template-lint . --fix",
     "lint:js": "eslint . --cache --ext=.js,.ts",
     "lint:js:fix": "eslint . --fix",
-    "lint:types": "tsc --noEmit",
+    "lint:types": "tsc --emitDeclarationOnly false --noEmit",
```

Comment:
End-developers can somewhat simplify the `lint:types` script if they have 2 TS configuration files, one for development and another for production.

```json5
/* package.json for projects without Glint */
"scripts": {
  "build:types": "tsc",
  "lint:types": "tsc --project tsconfig.development.json",
  "start:types": "tsc --watch"
}
```

```json5
/* tsconfig.json */
{
  "extends": "@tsconfig/ember/tsconfig.json",
  "compilerOptions": {
    "declaration": true,
    "declarationDir": "declarations",
    "emitDeclarationOnly": true,
    "noEmit": false
  },
  "include": [
    "src/**/*",
    "unpublished-development-types/**/*"
  ]
}
```

```json5
/* tsconfig.development.json */
{
  "extends": "@tsconfig/ember/tsconfig.json",
  "compilerOptions": {
    "declaration": true,
    "declarationDir": "declarations"
  },
  "include": [
    "src/**/*",
    "unpublished-development-types/**/*"
  ]
}
```

Since it's not likely that a v1 addon had used a second configuration file such as `tsconfig.development.json`, I didn't think that the codemod should create this file for end-developers.